includes/partials/admin-page.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * The admin page template.
+ *
+ * @package ICW
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+$post_types = get_post_types(
+	array(
+		'show_ui' => true,
+		'public'  => true,
+	),
+	'objects'
+);
+?>
+
+<div class="wrap">
+	<h2><?php esc_html_e( 'Insecure Content Warning', 'insecure-content-warning' ); ?></h2>
+
+	<h3><?php esc_html_e( 'Fix insecure content', 'insecure-content-warning' ); ?></h3>
+
+	<table class="form-table">
+		<tbody>
+			<tr>
+				<th scope="row"><label for="icw-post-selection"><?php esc_html_e( 'Post selection', 'insecure-content-warning' ); ?></label></th>
+				<td>
+					<select name="post_selection" id="icw-post-selection">
+						<option selected="selected" value="<?php echo esc_attr( 'all' ); ?>"><?php esc_html_e( 'All', 'insecure-content-warning' ); ?></option>
+						<option value="<?php echo esc_attr( 'posts' ); ?>"><?php esc_html_e( 'Individual post(s)', 'insecure-content-warning' ); ?></option>
+						<option value="<?php echo esc_attr( 'all_from_post_type' ); ?>"><?php esc_html_e( 'All from post type', 'insecure-content-warning' ); ?></option>
```

Comment:
No need to escape the constants, just variables and translations/

```suggestion
						<option selected="selected" value="all"><?php esc_html_e( 'All', 'insecure-content-warning' ); ?></option>
						<option value="posts"><?php esc_html_e( 'Individual post(s)', 'insecure-content-warning' ); ?></option>
						<option value="all_from_post_type"><?php esc_html_e( 'All from post type', 'insecure-content-warning' ); ?></option>
```

Reply:
Good point, thank you.